Diary/Domain.Entities/User.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Domain.Entities.BaseTypes;
+
```

Comment:
the same. On the whole, it will be resolved by switching from 'main' to 'develop' as target branch for PR.

Check all your files for merging and get rid of files which is not related with your microservice.